config/connection.js

```diff
@@ -5,7 +5,7 @@ var connection = mysql.createConnection({
   host: "localhost",
   port: 3306,
   user: "root",
-  password: "password",
```

Comment:
Look into npm install dotenv package , I think all you need to do after that is make a .env file to hide your password so its not technically pushed to github. Then add .env in .gitignore so its hidden. Other than that its great, and let Rick give a review of this